AirCasting/Graph/GraphDataGenerator.swift

```diff
@@ -0,0 +1,43 @@
+// Created by Lunar on 01/06/2021.
+//
+
+import Foundation
+
+class GraphDataGeneraator {
+    
+    private let defaultLimit = 1000
+    
+    private var cumulativeValue = 0
+    private var cumulativeMeasurementCount = 0
+        
+    
+    func generate(stream: MeasurementStreamEntity) -> [MeasurementEntity] {
+    
+        guard let allMeasurements = stream.allMeasurements else { return [] }
+        let measaurementCount = allMeasurements.count
+            
+        let fillFactor = Double(defaultLimit / measaurementCount)
+        var fill = 0.0
+        
+        for measurement in allMeasurements {
+            fill += fillFactor
+            
+            if fill > 1 {
+                fill = 0.0
+            }
+        }
+        
+        
+        
```

Comment:
Is the spacing really needed?

Reply:
There's no GraphDataGenerator anymore. I don't know why github didn't mark this comment as outdated.